example/README.md

```diff
@@ -31,6 +31,21 @@ To send requests from Ruby code, run the hello_world client example:
 bundle exec ruby hello_world_client.rb
 ```
 
+### Strict JSON Requests
+
+By default JSON requests ignore unknown fields to match the behavior of a
+traditional protocol buffer request.  In certain cases it can be convenient to
+have a request validated and fail if unknown fields are encountered --
+especially when doing testing using curl.  This can be done by specifying a
+strict flag as part of the content type.
+
+```sh
+curl --request POST \
+  --url http://localhost:8080/twirp/example.hello_world.HelloWorld/Hello \
+  --header 'Content-Type: application/json; strict=true' \
+  --data '{"name": "World", "unknown": "field will fail"}'
+```
```

Comment:
No need to include this part in the docs. Instead of that, let's add `strict=true` on all other examples. If all the examples with curl include `strict=true`, people will use that already by default. If strict mode is not used by default, then it is not that useful.

Reply:
Done

Reply:
I just updated the original curl example to include strict=true and removed my note and example